source/cloud-native-platform/guides/postgresql-singleserver-to-flexibleserver-migration-portal.html.md.erb

```diff
@@ -0,0 +1,29 @@
+---
+title: Postgresql single server to flexible server migration using Azure Portal
+last_reviewed_on: 2023-04-24
+review_in: 6 months
+weight: 10
+hide_in_navigation: true
+---
+
+# <%= current_page.data.title %>
+
+The Azure Portal can be used to perform an offline migration of a PostgreSQL Single Server instance to PostgreSQL Flexible Server.
```

Comment:
Just thinking that this will probably change as we'll have the pipeline to do this. If you leave this open I'll add some instructions around that